backend/pick-git/src/main/java/com/woowacourse/pickgit/post/application/PostService.java

```diff
@@ -102,21 +116,33 @@ private File tryCreateTempFile(MultipartFile multipartFile) {
         }
     }
 
-    public CommentResponseDto addComment(CommentRequestDto commentRequestDto) {
+    public CommentDto addComment(CommentRequestDto commentRequestDto) {
         User user = userRepository.findByBasicProfile_Name(commentRequestDto.getUserName())
             .orElseThrow(IllegalArgumentException::new);
         Post post = postRepository.findById(commentRequestDto.getPostId())
             .orElseThrow(IllegalArgumentException::new);
         Comment comment = new Comment(commentRequestDto.getContent());
         user.addComment(post, comment);
-        return new CommentResponseDto(user.getName(), comment.getContent());
+        entityManager.flush();
+        return CommentDto.from(comment);
     }
 
     @Transactional(readOnly = true)
     public RepositoriesResponseDto showRepositories(RepositoryRequestDto repositoryRequestDto) {
-        List<RepositoryResponseDto> repositories = platformRepositoryExtractor
-            .extract(repositoryRequestDto.getToken(), repositoryRequestDto.getUsername());
-
-        return new RepositoriesResponseDto(repositories);
+            List<RepositoryResponseDto> repositories = platformRepositoryExtractor
+                .extract(repositoryRequestDto.getToken(), repositoryRequestDto.getUsername());
+
+            return new RepositoriesResponseDto(repositories);
+     }
+
+    public List<PostDto> readHomeFeed(HomeFeedRequest homeFeedRequest) {
+        int page = Math.toIntExact(homeFeedRequest.getPage());
+        int limit = Math.toIntExact(homeFeedRequest.getLimit());
+        List<Post> result = entityManager
+            .createQuery("select p from Post p order by p.id", Post.class)
```

Comment:
```suggestion
            .createQuery("select p from Post p order by p.id desc", Post.class)
```

추후에 로그인 상태에서 피드 조회 시 게시글이 최신 순으로 나타나도록 변경해주세요 !